src/style/global.css

```diff
@@ -276,3 +276,8 @@ a[href].observablehq-header-anchor {
 .muted {
   color: var(--theme-foreground-muted);
 }
+
+.observablehq--draft > :first-child::before {
+  content: "[DRAFT] ";
```

Comment:
Slight preference to put it _after_ the title rather than before, and use muted instead of focus.

Reply:
Putting it after is not great if the first-child is (say) an image or a paragraph.

![image-after](https://github.com/observablehq/framework/assets/7001/203312f4-f320-4837-b298-c077968b6074)
![paragraph-after](https://github.com/observablehq/framework/assets/7001/a9f20f1e-34a8-4bde-9e59-d0bf7cf55e88)


and using first-child is probably not a great idea because… 🙃 (see screen cap below)

https://github.com/observablehq/framework/assets/7001/9a90e695-f14e-41f9-b755-9cdfc6984271

Reply:
Sorry, I assumed this was for `h1:first-of-type`, not `:first-child`. I think the [draft] should only apply to the h1.

Reply:
`.observablehq--draft > h1:first-of-type::after` it is!